spec/fixtures/files/datahashes/core/intake1.json

```diff
@@ -2,16 +2,16 @@
     "entryNumber": "IN2020.21",
     "entryDate": "2020-09-25",
     "entryReason": "consideration",
-    "entryMethod": "post;foundondoorstep",
```

Comment:
For vocabulary terms, input data should use the same form as between the single quotes at the end of the refname added to the cache